pallets/subtensor/src/lib.rs

```diff
@@ -313,103 +313,116 @@ pub mod pallet {
     /// ============================
     /// ==== Staking + Accounts ====
     /// ============================
-
-    #[pallet::type_value]
     /// Default value for zero.
+    #[pallet::type_value]
     pub fn DefaultZeroU64<T: Config>() -> u64 {
         0
     }
+
     /// Default value for Alpha currency.
     #[pallet::type_value]
     pub fn DefaultZeroAlpha<T: Config>() -> AlphaCurrency {
         AlphaCurrency::ZERO
     }
+
     /// Default value for Tao currency.
     #[pallet::type_value]
     pub fn DefaultZeroTao<T: Config>() -> TaoCurrency {
         TaoCurrency::ZERO
     }
-    #[pallet::type_value]
+
     /// Default value for zero.
+    #[pallet::type_value]
     pub fn DefaultZeroU128<T: Config>() -> u128 {
         0
     }
-    #[pallet::type_value]
+
     /// Default value for zero.
+    #[pallet::type_value]
     pub fn DefaultZeroU16<T: Config>() -> u16 {
         0
     }
-    #[pallet::type_value]
+
     /// Default value for false.
+    #[pallet::type_value]
     pub fn DefaultFalse<T: Config>() -> bool {
         false
     }
-    #[pallet::type_value]
+
     /// Default value for false.
+    #[pallet::type_value]
     pub fn DefaultTrue<T: Config>() -> bool {
         true
     }
-    #[pallet::type_value]
+
     /// Total Rao in circulation.
+    #[pallet::type_value]
     pub fn TotalSupply<T: Config>() -> u64 {
         21_000_000_000_000_000
     }
-    #[pallet::type_value]
+
     /// Default Delegate Take.
+    #[pallet::type_value]
     pub fn DefaultDelegateTake<T: Config>() -> u16 {
         T::InitialDefaultDelegateTake::get()
     }
```

Comment:
This file needs a cleanup because this makes no sense to wrap the `#[pallet::constant]` into a type value, we can just use it like that, `type_value` derives a `Get<T>` implementation that is already existing for `InitialDefaultDelegateTake`

Also, a lot of these type values are not used anywhere

edit: I added this to the #2047 issue